lib/travis/hub/model/log.rb

```diff
@@ -14,4 +14,12 @@ def clear
     update_column(:removed_by, nil)
     Part.where(log_id: id).delete_all
   end
+
+  def add(msg)
+    if aggregated_at
+      update_column(:content, [content, msg].join("\n"))
+    else
+      Part.create!(log_id: id, content: msg, final: true)
```

Comment:
@henrikhodne I was actually hoping you might be able to shed some light on this. I'm not 100% sure this is the right thing to do here. Would it be better to message Logs via HTTP? I guess the log could be in various different states at this point.